### PR TITLE
PAYARA-1255 enhance payaradomain template

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

File: appserver/admin/payara_template/src/main/resources/config/domain.xml [hunk 1]
```diff
@@ -169,26 +169,25 @@
...
         <jvm-options>-Djava.ext.dirs=${com.sun.aas.javaRoot}/lib/ext${path.separator}${com.sun.aas.javaRoot}/jre/lib/ext${path.separator}${com.sun.aas.instanceRoot}/lib/ext</jvm-options>
         <jvm-options>-Djdbc.drivers=org.apache.derby.jdbc.ClientDriver</jvm-options>
         <jvm-options>-DANTLR_USE_DIRECT_CLASS_LOADING=true</jvm-options>
         <jvm-options>-Dcom.sun.enterprise.config.config_environment_factory_class=com.sun.enterprise.config.serverbeans.AppserverConfigEnvironmentFactory</jvm-options>
-        <jvm-options>-XX:NewRatio=2</jvm-options>
         <jvm-options>-Xmx2g</jvm-options>
         <jvm-options>-Xms2g</jvm-options>
         <jvm-options>-Djdk.tls.rejectClientInitiatedRenegotiation=true</jvm-options>
-        <jvm-options>-Dorg.jboss.weld.serialization.beanIdentifierIndexOptimization=false</jvm-options>
+        <jvm-options>-Dorg.jboss.weld.serialization.beanIdentifierIndexOptimization=true</jvm-options>
```
Comment (mulderbaba) [inline appserver/admin/payara_template/src/main/resources/config/domain.xml]: this is already set to false due to #300.

**Comment by mulderbaba** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
I hadn't seen that, good spot

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by arjantijms** [comment]
@MattGill98 Seems intermediate changes caused conflicts, can you resolve these?

**Comment by MattGill98** [comment]
Branch got a bit messy, making a new PR

**Pull request closed without merging**